low-level/files/opt/seagate/sspl/conf/sample_global_cortx_config.yaml

```diff
@@ -52,3 +52,4 @@ rsyslog:
 release:
    product: LR2
    setup: cortx
+   target_build:
```

Comment:
i dont see this available in provisioner supplied confstor keys. Is it requested now ? We cant have code depending on keys which we may not come from provisioner !

Reply:
https://seagatetechnology-my.sharepoint.com/personal/shriya_deshmukh_seagate_com/Documents/Microsoft%20Teams%20Chat%20Files/provisioner_cluster.json
this is a file that was shared by provisioner, all confstor key-values config provided by provisioner mention here,
target build is present in release>target_build.

Reply:
Ok, i wasnt aware of this, was still referring to LR R2 Provisioner specfications sheet, where they list confstor keys which would be available.

Reply:
Code won't fail if the key does not exist. still, we can change it if the target_build key location is changed and confirmed by the provisioning team.

Reply:
alright. no issues.